src/coprocessor/codec/chunk/column.rs

```diff
@@ -0,0 +1,425 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::Result;
```

Comment:
please rearrange as use stds (not applicable) / use 3rd part libs (tipb::) / use other mods (util::) / use self mods (coprocessor::) / use parent or child mods (super::)

Reply:
It seems @BusyJay has adviced that we should order them by `rustfmt` now.